lib/api/core/indexCache.js

```diff
@@ -52,6 +53,16 @@ class IndexCache {
           promises.push(this.kuzzle.internalEngine.listCollections(index)
             .then(collections => {
               this.indexes[index] = collections;
+
+              const mappingPromises = collections.map(collection => {
```

Comment:
(Optional) This can be simplified: 

```js
const mapping = {
  properties: this.commonMapping
};

return Bluebird.map(collections, collection => this.kuzzle.internalEngine.updateMapping(collection, mapping, index));
```